bug-bounty-list/bug-bounty-list.json

```diff
@@ -7102,7 +6972,7 @@
   {
     "program_name": "Stackla",
     "policy_url": "https://support.stackla.com/hc/en-us/articles/115005937887-Reporting-a-possible-security-vulnerability-to-Stackla",
-    "submission_url": "",
+    "submission_url": "https://support.stackla.com/hc/en-us/articles/115005937887-Reporting-a-possible-security-vulnerability-to-Stackla",
```

Comment:
Looks like they just have a security@ inbox based on this description? You should probably avoid putting descriptive articles in this field (instead putting them in `policy_url`.